tests/phpunit/unit-tests/Model/TestModelPDFMetaBox.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace GFPDF\Model;
+
+use GFPDF\Controller\Controller_PDF;
+use GFPDF\Helper\Helper_Url_Signer;
+use GFPDF\View\View_PDF;
+use WP_UnitTestCase;
+
+/**
+ * @package     Gravity PDF
+ * @copyright   Copyright (c) 2020, Blue Liquid Designs
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ */
+
+/**
+ * Class TestModelPDFSuccess
+ *
+ * @package GFPDF\Model
+ *
+ * @group   pdf
+ */
+class TestModelPDFMetaBox extends WP_UnitTestCase {
+
+	/**
+	 * @var Controller_PDF
+	 */
+	protected $controller;
+
+	/**
+	 * @var Model_PDF
+	 */
+	protected $model;
+
+	/**
+	 * @var View_PDF
+	 */
+	protected $view;
+
+	/**
+	 * The WP Unit Test Set up function
+	 *
+	 * @since 4.0
+	 */
+
+	public function setUp() {
+		global $gfpdf;
+
+		/* run parent method */
+		parent::setUp();
+
+		/* Setup our test classes */
+		$this->model      = new Model_PDF( $gfpdf->gform, $gfpdf->log, $gfpdf->options, $gfpdf->data, $gfpdf->misc, $gfpdf->notices, $gfpdf->templates, new Helper_Url_Signer() );
+		$this->view       = new View_PDF( [], $gfpdf->gform, $gfpdf->log, $gfpdf->options, $gfpdf->data, $gfpdf->misc, $gfpdf->templates );
+		$this->controller = new Controller_PDF( $this->model, $this->view, $gfpdf->gform, $gfpdf->log, $gfpdf->misc );
+		$this->controller->init();
+	}
+
+	/**
+	 * Create our testing data
+	 *
+	 * @since 4.0
+	 */
+	protected function create_form_and_entries() {
+		global $gfpdf;
+
+		$form  = $GLOBALS['GFPDF_Test']->form['all-form-fields'];
+		$entry = $GLOBALS['GFPDF_Test']->entries['all-form-fields'][0];
+
+		$gfpdf->data->form_settings[ $form['id'] ] = $form['gfpdf_form_settings'];
+
+		return [
+			'form'  => $form,
+			'entry' => $entry,
+		];
+	}
+
+	/**
+	 * Check our PDF detail list is displaying correctly
+	 *
+	 * @since 4.0
+	 */
+	public function test_view_pdf_entry_detail_success() {
+
+		/* Setup some test data */
+		$results = $this->create_form_and_entries();
+
+		ob_start();
+		$this->model->view_pdf_entry_detail( $results );
+		$html = ob_get_clean();
+		$this->assertStringContainsString(  '<div class="gfpdf_detailed_pdf_cta">'  , $html );
```

Comment:
Code formatting incorrect. Run it through the linter.

Reply:
Copy and resolved